envs/vagrant/mirrors/group_vars/all.yml

```diff
@@ -46,3 +46,7 @@ apt:
     erlang:
       repo: http://173.247.104.68/packages.erlang-solutions.com/ubuntu
       key_url: http://173.247.104.68/keys/erlang.asc
+
+common:
+  pip:
+    proxy_url: http://173.247.104.68:8092/simple
```

Comment:
I added an empty common.pip.proxy_url to the defaults for common role. 

checked that pip is okay with that line being empty in the config file and it is.